### PR TITLE
Handle only the user workspace security settings

### PR DESCRIPTION
**Pull request by dhuebner**
#### What it does

Prevents setting workspace security settings in the workspace scope.

#### How to test

Open settings, switch to Workspace scope tab. `Security` should not be visible.
 

#### Review checklist

- [x] As an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)

#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

### CODE REVIEW

File: packages/workspace/src/browser/workspace-trust-service.ts [hunk 1]
```diff
@@ -115,17 +115,19 @@ export class WorkspaceTrustService {
     }
 
     protected async handlePreferenceChange(change: PreferenceChange): Promise<void> {
-        if (change.preferenceName === WORKSPACE_TRUST_STARTUP_PROMPT && change.newValue !== WorkspaceTrustPrompt.ONCE) {
-            this.storage.setData(STORAGE_TRUSTED, undefined);
-        }
+        if (change.scope === PreferenceScope.User) {
```
Comment (msujew) [inline packages/workspace/src/browser/workspace-trust-service.ts:118]: Suggestion: This change isn't necessary. The change event will not trigger if a `User` scoped preference is changed in the `Workspace` scope.

Comment (dhuebner) [reply packages/workspace/src/browser/workspace-trust-service.ts:118]: It's the second line of defense :)

Comment (msujew) [reply packages/workspace/src/browser/workspace-trust-service.ts:118]: Ok, fine with me.

**Review by msujew — APPROVED** [review]
Alright, works well for me 👍

**Pull request merged**